letsencrypt/cli.py

```diff
@@ -298,37 +298,103 @@ def _auth_from_domains(le_client, config, domains, plugins):
         # TREAT AS NEW REQUEST
         lineage = le_client.obtain_and_enroll_certificate(domains, plugins)
         if not lineage:
-            raise errors.Error("Certificate could not be obtained")
+            raise Error("Certificate could not be obtained")
 
     _report_new_cert(lineage.cert)
 
     return lineage
 
+def set_configurator(previously, now):
+    """Setting configurators multiple ways is okay, as long as they all agree"""
+    if now is None:
+        # we're not actually setting anything
+        return previously
+    if previously:
```

Comment:
`if previously is not None`? What is the meaning of those args anyway? Please update docstring with appropriate `:param:`. Maybe rename function to be private (`_set_configrator`)?